pages/project/read/views/index.jsx

```diff
@@ -309,7 +309,7 @@ export default function ProjectLandingPage() {
 
         <dt><Snippet>fields.licenceHolder.label</Snippet></dt>
         <dd>
-          {model.licenceHolder.firstName} {model.licenceHolder.lastName}<br />
+          {model.licenceHolder.name}<br />
```

Comment:
I would be wary about introducing dependency on the virtual property `name` existing. It's not there in certain contexts and should probably be removed generally.

If there's a reason why the old way didn't work then I'd go for this:

```suggestion
          {`${model.licenceHolder.firstName} ${model.licenceHolder.lastName}`}<br />
```